pennylane/templates/state_preparations/state_prep_mps.py

```diff
@@ -25,14 +27,20 @@ class MPSPrep(Operation):
 
     .. note::
 
-        Currently, this operator can only be used with ``qml.device(“lightning.tensor”)``.
+        Tensor simulators are designed to efficiently run MPS structures.
+
+        However, for non-`lightning.tensor` devices, a state-vector simulator may be useful if a gate decomposition is required.
+        This decomposition introduces work (auxiliary) qubits to prepare the MPS in a quantum circuit.
+
 
 
     Args:
         mps (List[Array]):  list of arrays of rank-3 and rank-2 tensors representing an MPS state as a
             product of site matrices. See the usage details section for more information.
 
         wires (Sequence[int]): wires that the template acts on
+        work_wires (Sequence[int]): list of extra qubits needed in the decomposition. The bond dimension of the mps
+                        is defined as ``2^len(work_wires)``. Default is ``None``
```

Comment:
Should we define this as the maximum permissible bond dimension of the provided MPS?